hoot-core/src/main/cpp/hoot/core/algorithms/aggregator/QuantileAggregator.h

```diff
@@ -47,13 +47,13 @@ class QuantileAggregator : public ValueAggregator, public Configurable
    */
   QuantileAggregator(double quantile);
 
-  virtual double aggregate(std::vector<double>& d) const;
+  virtual double aggregate(std::vector<double>& d) const override;
 
   virtual void setConfiguration(const Settings& conf);
```

Comment:
I wonder if this will also need `override` because it comes from the `Configurable` class.  Don't know if that will cause a SonarQube finding.  Just something to monitor in the next set of changes.